### PR TITLE
Example of using test event with node-lambda

### PR DESCRIPTION
**Pull request by kfriedman**
Not suggesting we necessarily merge this... just wanted to show this to you. Let's talk more!

### CODE REVIEW

File: tests/events/request-docs.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      "user": null,
+      "cognitoIdentityPoolId": null,
+      "cognitoIdentityId": null,
+      "cognitoAuthenticationProvider": null,
+      "sourceIp": "127.0.0.1",
+      "accountId": null
+    },
+    "stage": "prod"
+  },
+  "queryStringParameters": {
```
Comment (daryanypl) [inline tests/events/request-docs.json]: do these get filled out in later branches?  (sorry, not sure how jest works)

**Comment by LucasCioffi** [comment]
@kfriedman what should we look for in a response aside from a 200 status?
screen shot 2018-07-23 at 4 09 53 pm

File: tests/events/request-docs.json (hunk 1 above)
Comment (kfriedman) [reply tests/events/request-docs.json]: I guess it depends what you're testing. In this case, it's just testing the `docs/patron-creator` route with no query parameters.

**Comment by kfriedman** [comment]
@LucasCioffi I think a 200 response is a good start and, if you wanted, a string like `swagger` just to make sure it actually did what you would expect?

**Pull request merged**